.github/workflows/spotbugs.yml

```diff
@@ -0,0 +1,34 @@
+name: SpotBugs
+
+on:
+  pull_request:
+    branches:
+      - master
+    paths:
+      - 'powertools-core/**'
+      - 'powertools-logging/**'
+      - 'powertools-sqs/**'
+      - 'powertools-tracing/**'
+      - 'powertools-validation/**'
+      - 'powertools-parameters/**'
+      - 'powertools-metrics/**'
+      - 'pom.xml'
+      - '.github/workflows/**'
+jobs:
+  codecheck:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup java JDK 1.8
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'zulu'
```

Comment:
corretto ;)

Reply:
Unfortunately that is not supported by this action https://github.com/actions/setup-java#supported-distributions